src/android/ApptentiveBridge.kt

```diff
@@ -61,12 +61,19 @@ class ApptentiveBridge : CordovaPlugin(), ApptentiveActivityInfo {
 
           // Parse log level
           val logLevel = try {
-            args.getString(0)
```

Comment:
`cordova.exec(successCallback, errorCallback, "ApptentiveBridge", "deviceReady", [this.distributionVersion, loglevel]);`

So, is this the only place now execute is called and modified with the `distributionVersion`?

Reply:
Yes, `deviceReady` in the JavaScript file calls `registerWithLogs` using the `Info` log level, which calls the `deviceReady` native methods (which take distribution version as their first argument and log level as the second).